content/changelog/2024-07-12.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Update CLI including app bootstrapping, more Branches page improvements, and better safety around deleting and resetting branches
+---
+
+### Updates to the Neon CLI
+
+We've added some terrific new features to our CLI:
+
+- **Configurable compute sizing with `-cu` option, including support for autoscaling**
+
+  You can now set your compute size when creating a branch using the CLI. For a fixed compute size, use a single number (e.g., `--cu 2`). For autoscaling, specify a range with a dash (e.g., `--cu 0.5-3`).
+
+  **Example:**
+
+  ```bash
+  neon branches add-compute main --cu 0.5-3
+  ```
+
+  Previously, compute size was determined by your default settings in the console.
+
+- **Bootstrap new applications with Neon using the `create-app` command**
+
+  You can now scaffold applications using the CLI with the `create-app` command. Choose your package manager, framework, ORM, and authentication method, and go. This exciting new feature is just getting started. More to come!
+
+### Added polish to our Branches page
+
+Branching is still a relatively new concept in the database world, and we want to make your experience with it as intuitive as possible. Here are our latest updates to the **Branches** page:
+
+- Made it easier to create child branches by adding a clear-as-day **Create child branch** button to the detailed Branches view. It's still an option under the Actions dropdown, but now you've got the can't miss button too.
+  ![added create child branch button](/docs/relnotes/branches_create_child_branch.png)
+- Easier navigation from your listed database on **Roles and databases** tab to the Drizzle Studio-based **Tables** page, where you can explore and modify your data directly. The actions kebab also includes options to delete your database, as well as navigate to the SQL Editor, and we preserve your database selection as you travel.
+  ![databases edit and new kebab](/docs/relnotes/databases_actions.png)
+- We've changed the metric displayed on the Branches from **active time** to the more helpful **compute time** metric, giving you a better sense of how much compute resources a given branch is consuming. For more details about consumption metrics, see [Usage metrics](/docs/introduction/usage-metrics).
+
+### Even more protection
+
+Last week, we introduced protection against accidentally deleting protected branches. This week, we've added more safeguards:
+
+- **Default Branches:** We've made it so you can't accidentally delete your default branch. If you need to delete that branch, make another branch the default first.
```

Comment:
"We've made it so you can't accidentally delete your default branch." - this was true before, so this item can be removed.

Reply:
done